### ISSUE TITLE
Component folder should probably be named system

### ISSUE THREAD

**Issue by gabrielgiordano** [dry-rb/dry-web-roda#20]
Hi guys, I was trying to use dry-web-roda and I run into an issue that files inside the folder `component/boot` were not being initialized. I think it is because either we should change the config inside the container file, or we should rename the folder name to system.

Steps to reproduce the issue:

`dry-web-roda new issue`
Add a breaking point before `Issue::Container::finalize!`

``` ruby
# component/boot.rb
binding.pry
Issue::Container.finalize! do |container|
end
# ...
```

back to bash:
`bin/console`

At break point:

```
[2] pry(main)> Issue::Container.boot :logger
TypeError: no implicit conversion of Pathname into String
from /home/gabriel/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/dry-system-0.5.1/lib/dry/system/booter.rb:43:in `require'
[3] pry(main)> Issue::Container.finalize!
=> Issue::Container
[4] pry(main)> Issue::Container["logger"]
Dry::Container::Error: Nothing registered with the key "logger"
from /home/gabriel/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/dry-container-0.5.0/lib/dry/container/resolver.rb:22:in `block in call'
```

Adding `config.system_dir = "component"` to the container seems to fix the issue.
Maybe the folder should be named `system` because that is the default value from `Dry::System::Container`

https://github.com/dry-rb/dry-system/blob/master/lib/dry/system/container.rb#L69

If you guys need some help, I would be glad to help fix this issue, I just need some guidance since I never made a contribution to an open source project before and I don't know the code base here. 
Regards

**Comment by timriley** [comment]
Thanks for reporting this! Would love it if you could help with a fix :) I think we'd just want to rename our "component" dirs in the skeletons to "system", so it matches the new dry-system default. Would you be up for doing this?

**Comment by andriytyurnikov** [comment]
side note: guys, please consider bumping versions after merging fixes - working cli generator is very desired feature, by those, interested in checking advantages of dry-rb ecosystem ;)

**Comment by timriley** [comment]
@gabrielgiordano I'll take care of this one now.

**Comment by timriley** [comment]
Fixed in 0.5.0

**Comment by timriley** [comment]
Sorry about that, I'll work to be better on this.
